desub-legacy/src/test_suite.rs

```diff
@@ -25,6 +25,7 @@ pub fn mock_runtime(num: u32) -> RuntimeVersion {
 		spec_version: num,
 		impl_version: num,
 		apis: Cow::from(Vec::new()),
-		transaction_version: 4,
```

Comment:
Why changed? 

Does scale-value need newer metadata or something?

Reply:
Oh, I just didn't like that it was 4, which is identical to the version of the tx protocol (which is a different version!), so I set it to the version from polkadot iirc